app/controllers/katello/concerns/api/v2/registration_commands_controller_extensions.rb

```diff
@@ -28,7 +28,7 @@ module ApiPieExtensions
         update_api(:create) do
           param :registration_command, Hash do
             param :activation_key, String, desc: N_('Activation key for subscription-manager client, required for CentOS and Red Hat Enterprise Linux. For multiple keys use `activation_keys` param instead.'), deprecated: true
-            param :activation_keys, Array, desc: N_('Activation keys for subscription-manager client, required for CentOS and Red Hat Enterprise Linux. Required only if host group has no activation keys.')
+            param :activation_keys, Array, desc: N_('Activation keys for subscription-manager client, required for CentOS and Red Hat Enterprise Linux. Required only if host group has no activation keys or if you do not provide a host group.')
```

Comment:
I think "required for CentOS and RHEL" and "required only if host group has no activation keys" could be somewhat confusing, if not contradictory, statements. Can you think of an alternative word to replace the second instance of "required"? For example:

```suggestion
            param :activation_keys, Array, desc: N_('Activation keys for subscription-manager client, required for CentOS and Red Hat Enterprise Linux. Use only if host group has no activation keys or if you do not provide a host group.')
```

Reply:
There are two instances where `activation_keys` is not required:

1. The host is not CentOS or RHEL; or
2. The host group already provides an activation key.

So maybe 'required' is fine for both? I'm okay with 'use' as well